packages/veto-backend/src/api/cyphernode/watch.ts

```diff
@@ -0,0 +1,20 @@
+import { AxiosInstance, AxiosResponse } from 'axios'
+import cyphernodeClient from './client'
+
+const cyphernode: AxiosInstance = cyphernodeClient()
+
+type Args = {
+  address: string
+  eventMessage?: string
+}
+
+export default function watch({ address, eventMessage }: Args): Promise<AxiosResponse> {
+  return cyphernode.post('/watch', {
+    address,
+    // If running cyphernode locally use 'host.docker.internal'
+    // if using the the aws instance of cyphernode, you must use your IP.
+    unconfirmedCallbackURL: 'host.docker.internal:8080/api/notificationcallback',
```

Comment:
Since we plan on changing the domain based on environment, add an entry to `config.ts` so we can specify the domain with an environment variable.

Reply:
good call.  done.